osf/models/preprint_service.py

```diff
@@ -199,6 +200,10 @@ def set_published(self, published, auth, save=False):
             self.node.save()
             self.save()
 
+            # Send creator confirmation email
+            project_signals.contributor_added.send(self.node, contributor=auth.user, auth=auth,
```

Comment:
We chatted about this a bit, but adding notes here so I don't forget:

1. This should probably just be a send_mail. It looks like contributor added hits a couple places, including modifying subscription settings (see [this section](https://github.com/CenterForOpenScience/osf.io/blob/develop/website/notifications/listeners.py#L20))
2. If not a send_mail it can probably just be its own signal, but I'm not 100% clear on when signals should be used so understanding that better is probably necessary
3. See [this pr](https://github.com/CenterForOpenScience/osf.io/pull/7515/files#diff-51eef6ea2a6c35e2b080c9a6fe591e38R1636) for a pretty simple example of a sendmail